app/Resources/assets/sass/styles.scss

```diff
@@ -131,3 +131,8 @@ span.notification-url-indicator {
     margin-right: 15px;
     text-decoration: underline;
 }
+
+.btn-default.btn-reject{
```

Comment:
Bootstrap already have a `btn-danger` class to make the button red ( https://www.w3schools.com/bootstrap/bootstrap_buttons.asp for reference) so there is no need to manually create a new style, if you have an specific requirement to implement it this way, please ignore this comment. If not please use `btn-danger` instead. This way we can be consistent with the style guide